docs/sources/flow/reference/cli/run.md

```diff
@@ -35,6 +35,7 @@ The following flags are supported:
 * `--storage.path`: Base directory where components can store data (default `data-agent/`).
 * `--disable-reporting`: Disable [usage reporting][] of enabled [components][] to Grafana (default `false`).
 * `--cluster.enabled`: Start the Agent in clustered mode (default `false`).
+* `--cluster.node-name`: The name to use for this node (defaults to the environment's HOSTNAME).
```

Comment:
Is the all caps semantically meaningful here? You use HOSTNAME here but "hostname" down below. Since all caps usually indicate placeholders, maybe this should be "hostname" instead.

@clayton-cornell WDYT?

Reply:
Putting this in lowercase. It'd make more sense as `$HOSTNAME` but I don't really like that.